uk.co.compucorp.civicrm.hrcontactactionsmenu/hrcontactactionsmenu.php

```diff
@@ -120,4 +120,23 @@ function hrcontactactionsmenu_civicrm_angularModules(&$angularModules) {
  */
 function hrcontactactionsmenu_civicrm_alterSettingsFolders(&$metaDataFolders = NULL) {
   _hrcontactactionsmenu_civix_civicrm_alterSettingsFolders($metaDataFolders);
+}
+
+/**
+ * Implementation of hook_civicrm_pageRun
+ */
+function hrcontactactionsmenu_civicrm_pageRun(&$page) {
+  CRM_Core_Region::instance('contact-page-inline-actions')->update('default', array(
+    'disabled' => TRUE,
+  ));
+  $template_path = CRM_Core_Resources::singleton()->getPath('uk.co.compucorp.civicrm.hrcontactactionsmenu','templates/CRM/ContactActionsMenu/Page/Inline/ActionsPart2.tpl');
+  
+  if ($page instanceof CRM_Contact_Page_View_Summary) {
+    CRM_Core_Region::instance('contact-page-inline-actions')->add(array(
+      'template' => $template_path,
+    ));
+  }
+  
```

Comment:
@bmnepali , I would suggest you remove the code in the hrcontactactionsmenu_civicrm_pageRun above and only leave the part that adds the css file to the template. 
The reason is that we would be overriding the Actions template in the Summary.tpl file of the HRUI extension and not via CrmRegion tag again as discussed in the last contact actions catch up call. 
I would add the code to implement this as part of https://compucorp.atlassian.net/browse/PCHR-3088.
Also when adding the css style wrap it in the condition like this:

```php
if ($page instanceof CRM_Contact_Page_View_Summary) {
  $extName = 'uk.co.compucorp.civicrm.hrcontactactionsmenu';
  CRM_Core_Resources::singleton()->addStyleFile($extName, 'css/contactactions.css');
}
```